### PR TITLE
Ung 1045/replace sqs with kafka for trackle

### PR DESCRIPTION
**Pull request by Hiroki6**

### CODE REVIEW

File: config/src/main/resources/application.base.conf [hunk 1]
```diff
@@ -31,6 +31,15 @@ akka {
       connecting-timeout = 2s
     }
   }
+  kafka {
+    producer {
+      kafka-clients {
+        acks = 1
+        delivery.timeout.ms = 60000 // 60 seconds
```
Comment (Hiroki6) [inline config/src/main/resources/application.base.conf:38]: actor timeout should be set to the same or longer value as this value in the prod configuration.

File: config/src/main/resources/application.base.conf [hunk 2]
```diff
@@ -31,6 +31,15 @@ akka {
       connecting-timeout = 2s
     }
   }
+  kafka {
+    producer {
```
Comment (Hiroki6) [inline config/src/main/resources/application.base.conf:35]: Regarding maximum batch size, the size of a msgPack is `48,144` bytes.
The default maximum batch size is `1048576`, so probably no problem.

File: core/src/main/scala/com/ubirch/transformer/actor/KafkaProducerActor.scala [hunk 3]
```diff
@@ -0,0 +1,35 @@
+package com.ubirch.transformer.actor
+
+import akka.actor.{Actor, ActorLogging, Props}
+import akka.pattern.pipe
+import com.ubirch.avatar.core.kafka.KafkaProducer
+import com.ubirch.transformer.actor.KafkaProducerActor.KafkaMessage
+
+import scala.concurrent.ExecutionContext
+
+class KafkaProducerActor(kafkaUrl: String, topic: String) extends Actor with ActorLogging {
```
Comment (Hiroki6) [inline core/src/main/scala/com/ubirch/transformer/actor/KafkaProducerActor.scala:10]: I implemented Kafka part as an Actor according to the consistency of this service, which is that each actor has a responsibility of one operation such as MessagePersistenceActor, TransformerOutProducerActor.

File: config/src/main/resources/application.docker.conf [hunk 4]
```diff
@@ -87,7 +96,7 @@ ubirchAvatarService {
   }
 
   akka {
-    actorTimeout = 10
+    actorTimeout = ${AKKA_ACTOR_TIMEOUT} // seconds
```
Comment (lalottacontinua) [inline config/src/main/resources/application.docker.conf:99]: did you adjust the helm charts accordingly already? But good idea to externalise that config.

Comment (Hiroki6) [reply config/src/main/resources/application.docker.conf:99]: I will adjust later with the configuration of trackle serivce.

**Pull request merged**